src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+import duke.commands.Command;
+import duke.others.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.storage.TaskList;
+
+/**
+ * Represents a bot that will carry out different command with user input.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+
+    /**
+     * Constructs a new Duke instance,
+     * and loads the content of the local file into storage.
+     *
+     * @param filePath path of the local path.
+     */
+    public Duke(String filePath) {
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (DukeException e) {
+            tasks = new TaskList();
+        }
+    }
+
+    /**
```

Comment:
Just a suggestion, maybe you could remove these methods if you do not need them?